### PR TITLE
Add project popup feature

### PR DESCRIPTION
**Pull request by elassis**
What I Have Accomplished 🏆 : 

* Use javascript to create a popup of each project on the website
* Apply CSS properties to give the appearance similar to the mockup template in Figma.

# Built With ⚒️ : 
* CSS
* HTML
* Javascript

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,4 +6,244 @@ function showMenu() {
...
+    'image-project-3',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+  [
+    'multipost-stories',
+    'elements/snapshoot-4.png',
+    'image-project-4',
+    'A daily selection of privately personalized reads no accounts or sign-ups required.',
+  ],
+];
```
Comment (MahmoudBakr23) [inline script.js:64]: - [ ] **OPTIONAL**
Great job managing to code the array of objects, but don't you think it's better when you wrap the objects directly to the array! Please tell me what you think 😄

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
_If you think about it, it'll shorten a lot of lines of code, and less code is better anywhere and anytime. 🎉_

File: style.css [hunk 2]
```diff
@@ -147,6 +152,14 @@ ul.work {
   height: 10px;
 }
 
+.popup-img > img {
+  width: 100%;
```
Comment (MahmoudBakr23) [inline style.css:156]: - [ ] The image is of lower quality with the full width and height, if you tried to shrink it a little bit as in the given design, it's going to look better 😉 
- [ ] Also, please fix the modal's styling on the mobile screen to match the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
### **Your design**
Screenshot

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @elassis 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Linter is correct 💯 
- Descriptive PR summary and headline 💯 
- Good use of JS logical code and DOM manipulations so far 👍

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @elassis  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**